src/components/draw/feature-measurements.vue

```diff
@@ -136,7 +140,9 @@ function onClickValidateRadius(radius: number) {
     <!-- Feature elevation, for Point -->
     <div v-if="featureType === 'drawnPoint'">
       <span>{{ t('Elevation') }}: </span>
-      <span data-cy="featItemElevation" v-format-length="featElevation"></span>
+      <span data-cy="featItemElevation" v-format-measures.length>{{
+        featElevation
+      }}</span>
```

Comment:
Thanks for the refactoring! I think we could use your new elevation type for the directive here.

Reply:
yes, done